sim/src/main/cc/fasedtests/fasedtests_top.cc

```diff
@@ -76,7 +75,7 @@ bool fasedtests_top_t::simulation_complete() {
 }
 
 uint64_t fasedtests_top_t::profile_models() {
-  for (auto &mod : registry.get_all_models()) {
+  for (auto &mod : registry.get_bridges<FASEDMemoryTimingModel>()) {
```

Comment:
This is out of scope for this PR, but do you think we should just have a flag on MMIO registers in widget that basically say "i am instrumentation, please profile me". Then the profiling system could trivially apply to all bridges without adding any new driver code.